aws/scripts/sh/rotate_github_token.sh

```
@@ -0,0 +1,50 @@
+installation_id=$(echo "$response" | jq -r '.[0].id')
+
+if [ -z "$installation_id" ] || [ "$installation_id" = "null" ]; then
+  echo "Failed to retrieve installation ID"
+  exit 1
+fi
+
+# Create an installation access token
+token_response=$(curl -s -X POST \
+  -H "Authorization: Bearer $jwt" \
+  -H "Accept: application/vnd.github.v3+json" \
+  https://api.github.com/app/installations/$installation_id/access_tokens)
```

Shellcheck

[info] Double quote to prevent globbing and word splitting.

(SC2086)